packages/algolia/src/client.ts

```diff
@@ -88,6 +91,7 @@ export class AlgoliaSearchClient {
     entityTypes: T[],
     query: string,
     requestOptions?: SearchOptions,
+    descending?: boolean,
```

Comment:
Piotr: what if we need another replica for another different sort type? Are these replicas there just so we can search in reverse order? I think other combinations might be required later, maybe we need some sort of map here for sort order? also technically at the moment this replica should only ever be used for events - shouldnt our type or implementation reflect that?

Reply:
@peterstarling Yeah the replicas are the only way to sort, if we do need other combinations later, from what I gather that would mean having multiple replica indexes and we will have to refactor the search function a bit I think.

Reply:
i think what i was trying to say here was that the interface seems a little poor. On one hand I get it - for now all we need is a parameter to tell us if its descending or not but in reality it should be a little more sophisticated, type-wise. For instance we should not be able to select just any entity type here, just the events, as this is what has been currently sorted in a descending order, correct?

Reply:
i think we said we should abstract away the interface for search anyway here so I would suggest creating a ticket to do that where we also make the order parameter in a less concrete way